scripts/install/install-windows.ps1

```diff
@@ -122,8 +122,8 @@ function install-python-dependencies() {
         add-failure "Invalid version of python installed. Needs python >= 3.5. You can either have globably available in the path as python, python3 or set the BL_PYTHON_PATH environment variable."
     }
 
-    Write-Host "$python $root/scripts/install/install-python-dep.py"
-    cmd /c "$python $root/scripts/install/install-python-dep.py"
+    Write-Host "$python `"$root/scripts/install/install-python-dep.py`""
```

Comment:
The quoting on this and the next line looks wrong to me.

Reply:
@gingi This PR looks fine to me minus the strange quoting here - mind taking a look?